lib/execution_engine2/db/models/models.py

```diff
@@ -312,15 +313,22 @@ class Job(Document):
 
     terminated_code = IntField(validation=valid_termination_code)
     error_code = IntField(validation=valid_errorcode)
-
+    batch_job = BooleanField(default=False)
     scheduler_type = StringField()
     scheduler_id = StringField()
     scheduler_estimator_id = StringField()
     job_input = EmbeddedDocumentField(JobInput, required=True)
     job_output = DynamicField()
     condor_job_ads = DynamicField()
-    child_jobs = ListField()
-    batch_job = BooleanField(default=False)
+    child_jobs = ListField()  # Only parent container should have child jobs
+    # batch_parent_container = BooleanField(default=False) # Only parent container should have this
+
+    # Only present when a job has been retried and on the retry_parent
+    retry_count = IntField(min_value=0)
+    retried = BooleanField()
```

Comment:
Now I think about it isn't the `retried` field redundant? It's equivalent to `retry_count > 0`, right?

Reply:
Yes, lets get rid of it

Reply:
:+1: